### ISSUE TITLE
Stats command

### ISSUE THREAD

**Issue by super-cooper** [super-cooper/memebot#3]
**Is your feature request related to a problem? Please describe.**
N/A

**Describe the solution you'd like**
For now, a basic `!stats` command that mimics the current stats bot that we use. For holding on to this data, I think MongoDB would be a handy solution because 
1. None of us know how to make a real database
2. Python integrates very well with JSON structure

**Describe alternatives you've considered**
A more feature-rich stats command with more advanced statistics and advanced queries may come down the line.

**Additional context**
N/A

**Comment by nhawke** [comment]
Maybe not worth implementing until #13 is done, but a dumb/slow version of this could just count messages on-demand rather than track stats in a database. Doing it ths dumb way would prevent DB annoyances of either needing to update the DB constantly or keep all our messages in the DB (although the frequent small overhead of updating the DB constantly might be preferred to the large cost of counting up everything on demand)

**Comment by super-cooper** [comment]
>Maybe not worth implementing until #13 is done, but a dumb/slow version of this could just count messages on-demand rather than track stats in a database.

Agreed. I was planning on making some sort of DB interface and we could just hook commands and routines up to it.
